src/mlpack/methods/ann/activation_functions/sqnl_function.hpp

```diff
@@ -0,0 +1,143 @@
+/**
+ * @file sqnl_function.hpp
+ * @author Prince Gupta
+ *
+ * Definition and implementation of the SQNL(Square Non Linearlity)
+ * activation function.
+ *
```

Comment:
Could you also add reference to the paper from here, https://ieeexplore.ieee.org/document/8489043 ( I think this is the right paper).